tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite with ICs and LBCs derived from the RRFS forecasts mapped onto
+    3-km regular grid (rrfs*.conus.grib2). This test uses old v1 sfc_data,
+    not the v2 fractional grid sfc_data.
+user:
+  RUN_ENVIR: community
+workflow:
+  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
+  PREDEF_GRID_NAME: RRFS_CONUScompact_25km
+  DATE_FIRST_CYCL: '2024060517'
+  DATE_LAST_CYCL: '2024060517'
+  FCST_LEN_HRS: 9
+  PREEXISTING_DIR_METHOD: rename
+task_make_orog:
+  OMP_NUM_THREADS_MAKE_OROG: 24
+task_get_extrn_ics:
+  EXTRN_MDL_NAME_ICS: RRFS
+  FV3GFS_FILE_FMT_ICS: grib2
+  USE_USER_STAGED_EXTRN_FILES: true
+task_get_extrn_lbcs:
+  EXTRN_MDL_NAME_LBCS: RRFS
+  LBC_SPEC_INTVL_HRS: 1
+  FV3GFS_FILE_FMT_LBCS: grib2
+  USE_USER_STAGED_EXTRN_FILES: true
+task_plot_allvars:
+  COMOUT_REF: ""
+global:
+  DO_ENSEMBLE: true
+  NUM_ENS_MEMBERS: 3
```

Comment:
Why do we need an ensemble for this test? It's just a grid/ICS/LBCS test.

Reply:
This PR includes functionality of having ensembles with plotting tasks made for each ensemble. It will be needed if we use (later) different physics for the ensembles.

Reply:
Adding a feature with different ensemble members running different physics scheme was marked as a feature for RRFS integration. This test case is developed to combine different testing of these features: current features are i) use of any hour of the day, ii) use of RRFS ICS/LBCS, and the subsequent feature would be iii) use of different physics for the ensemble members. It appears that having a single case for RRFS to test, modify, and explore capabilities would be easier to track for the SRW code modification, rather than attempting to modify existing cases in addition to a specific RRFS test case. But I'd be OK either way.

Reply:
How expensive is this new, additional ensemble? If I'm not mistaken, 3 members is the largest ensemble we have in the test suite. Would graphics need to run on all three members for a valid test? Given that this isn't currently a multi-physics ensemble test, why do we need to spend the resources now for something that has not yet been developed. That could potentially be weeks before we see the need for another ensemble test.

Reply:
The number of ensembles could be set to 2, and forecast to 3 hours, in  to minimize use of resources. This test has been added to a comprehensive suite, which may not not need to be run for each and every PR (fundamental tests are run for each PR)
A grid being tested is 25-km CONUS compact grid, and is a relatively inexpensive option.

The cost of this experiment with 3 ensemble members and 9-h forecast: 
51.06 core-hours ([link to a comment on timing](https://github.com/ufs-community/ufs-srweather-app/pull/1089#pullrequestreview-2130788005)). 
- No other tests are run with 3 ensemble members; 
- Forecast duration set to cover timing of tornadoes reported over Maryland plus preceding 3 hours for model spin-up
 
The cost of this experiment with 2 ensemble members and 3-h forecast:
33.85 core-hours.

Maybe @MichaelLueken could weigh in on the option to cut the ensemble size and forecast extent?

This test includes implementation for running graphic for individual ensemble members, which was not available earlier. Thus yes, running graphics for ensemble member is needed.

Reply:
@natalie-perlin, you decided on this date due to a significant weather event that was taking place at the time.  Is a 9-h forecast required to ensure that the event is in the forecast window?  Would either a 3-h or 6-h forecast work for this?  If so, then decreasing the forecast length to ensure that the event is forecasted would be best.  As for the ensemble size, decreasing it to 2 members until the capability to run different physics per ensemble is introduced is fine.  Out of curiosity, which physics are being planned for this future work?

Reply:
Yes, this date was chosen for a significant weather event. The target forecast of 9-hours was intended to include 3-h spinup prior to the first reported tornado event, and covers timing of the last report. And yes, reducing ensemble size to 2 works OK, if having one 3-ensemble test does not add any benefits to testing features. For the current 25-km grid we do not expect to have any notable tornado features. Developing this test further to have a finer 3-km or 1-km grid over the affected areas may have more sense to have the forecast time extended to 9 hours. 
  So yes, I'd update the current test to contain 2 ensembles and 3-h forecast extent.

Reply:
As to physics, I've tested FV3_HRRR and RRFS_v1beta so far, and would want find at least one or two more that work well.

Reply:
... done! Ensemble size and forecast period reduced for the new test.